gregBorchers/HW5Problem1.html

```diff
@@ -0,0 +1,10 @@
+<html>
```

Comment:
What's here looks good so far. I've commented on part of problem 3 that could use a little tweaking, but other than that it's solid stuff. Your `console.log`s are great&mdash;very easy to see what's going on in your code.